stacks-signer/src/tests/conf/signer-0.toml

```diff
@@ -0,0 +1,19 @@
+
+message_private_key = "CosqumxnRfykP24DdUCDEEMZ2qCwHPUyBy21Qf55EPKJ"
+stacks_private_key = "8a99f02ddede302a0e9fbb4eb4ef1bd587a9c076f67230c3e7a62a9d78d34d0401"
+node_host = "127.0.0.1:20444"
```

Comment:
Port 20444 is usually the p2p interface.  The RPC interface is usually on 20443.  Is this intentional here?

Reply:
Naw this was not intentional. Was just the first port I thought of when I called the cli command to generate the files.